src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -68,12 +92,27 @@ public void visitToken(DetailAST arrayInit) {
 
         // if curlies are on the same line
         // or array is empty then check nothing
-        if (arrayInit.getLineNo() != rcurly.getLineNo()
+        if ((!ignoreInlineArrays || arrayInit.getLineNo() != rcurly.getLineNo())
```

Comment:
Please, split the condition.

```java
if (condition1
    || condition2) {
}
```

it will ease resolving the coverage problem for you and for the future contributors.